framework/CHANGELOG.md

```diff
@@ -9,12 +9,13 @@ Yii Framework 2 Change Log
 - Enh #12938: Allow to pass additional parameters to `yii\base\View::renderDynamic()` (mikehaertl)
 - Enh #13006: Added a `/` to the `yii\captcha\Captcha::$captchaAction` string to work correctly in a module also (boehsermoe)
 - Removed methods marked as deprected in 2.0.x (samdark)
+- Chg #7770: Updated the fallback date formats used when no `intl` extension is installed to match the defaults of the latest ICU version (cebe)
```

Comment:
No, it's OK.

> Additional fixes:
> - Reordered changes for 2.1 in CHANGELOG.

I reordered changes for 2.1 to preserve order in dedicated sections like it was done in 2.0.11. I thinj we need to add a note about modifying a changelog according to that and be more accurate when reviewing PRs.